### DISCUSSION TITLE
Add tag to QGIS plugin for "solar" or similar to aid indexing

### DISCUSSION THREAD

**Discussion by cmarmstrong** [UMEP-dev/UMEP#628] [General]
Interest in solar energy is accelerating and potential users may never find this wonderful plugin because it contains no tag referencing its solar analysis facilities, nor even much text in the description--other than "shadows"--that would help search engines find this.

Given the special interest that is accruing around solar, could a keyword for solar be added at least to the plugin tags to help other users find this gem?

**Comment by biglimp** [comment]
Good idea. Except from "solar", any other tags that we should add?

**Comment by biglimp** [comment]
added solar energy
